src/orient/rational-calc.orient

```diff
@@ -0,0 +1,354 @@
+
+    ProofGeneration:
+      describe(post_proof_gen, "Time to generate a PoSt Proof (including SNARKs, generating witnesses and reading from storage the challenged leaves)")
+      describe(post_proof_gen_parallel, "Time to generate a PoSt Proof (including SNARKs, generating witnesses and reading from storage the challenged leaves) in parallel")
+
+      post_proof_gen = post_snark_time + post_witness_gen 
+
+  Time (NoWrapper):
+    //rig_storage_latency_throughput_kb = rig_storage_latency_throughput / 1024
+    //response_honest_min = (post_challenges * challenged_sectors_percentage * rig_sectors_min * rig_storage_latency *windows) / rig_storage_latency_throughput
+
+  SNARK:
+    describe(inclusion_proof, "Constraints for checking an inclusion proof in a binary merkle tree as large as the sector")
+    describe(post_inclusion_proofs, "Constraints for checking all inclusion proofs (in PoSt)")
+    describe(ticket_proofs, "Constraints for checking the partial ticket (in PoSt)")
+    describe(ticket_constraints, "Constraints of a partial ticket per challenged node (in PoSt)")
+    describe(post_snark_constraints, "Constraints for a PoSt")
+    describe(post_snark_partitions, "Number of partitions of a PoSt proof")
+    describe(post_snark_size, "Size of a PoSt proof (bytes)")
+    describe(post_snark_size_kb, "Size of a Post proof (KiB)")
+    describe(post_snark_partition_constraints, "Number of constraints for a PoSt partition")
+
+    inclusion_proof = merkle_tree_hash_constraints * log2(nodes)
+    post_inclusion_proofs = inclusion_proof * post_challenges
+    ticket_proofs = ticket_constraints * post_challenges
+
+    post_snark_constraints = ticket_proofs + post_inclusion_proofs
+    post_snark_partitions = post_snark_constraints / post_snark_partition_constraints
+    post_snark_size = post_snark_partitions * snark_size
+    post_snark_size_kb = post_snark_size / 1024
+
+ProofOfReplication:
+  Security:
+    describe(porep_challenges, "Number of challenges in a PoRep proof")
+    describe(porep_lambda, "Bits of security of a PoRep proof")
+    describe(spacegap, "Maximum difference in storage between an honest prover and a maliciois prover")
+
+    porep_challenges = -porep_lambda / (log2(1-spacegap/4))
+
+  Graph:
+    describe(nodes, "Number of nodes in a sector")
+    describe(sector_size, "Size of a sector (bytes)")
+    describe(sector_size_gb, "Size of a sector (GiB)")
+
+    describe(window_nodes, "Number of nodes in a window")
+    describe(window_size, "Size of a window (bytes)")
+    describe(window_size_mb, "Size of a window (MiB)")
+    describe(windows, "Number of windows in a sector")
+
+    describe(graph_parents, "Number of parents of a node in the PoRep window graph")
+    describe(drg_parents, "Number of DRG parents of a node in the PoRep window graph")
+    describe(expander_parents, "Number of Chung Expander parents of a node in the PoRep window graph")
+    describe(drg_layers, "Number of layers in the DRG")
```

Comment:
This naming is a little confusing. The way I see it, there are the key layers, and the wrapper layers (just one). Each of those layers might contain a single DRG. So what you are describing is what I would call a key layer. I'm not saying it's wrong, just surfacing some terminology friction. I do think viewing a DRG as one layer is consistent with names like `drg_parents`.